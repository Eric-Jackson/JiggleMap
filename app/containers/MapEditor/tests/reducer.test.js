
import mapEditorReducer from '../reducer';

describe('mapEditorReducer', () => {
  it('returns the initial state', () => {
    expect(mapEditorReducer(undefined, {})).toMatchSnapshot();
  });
});
