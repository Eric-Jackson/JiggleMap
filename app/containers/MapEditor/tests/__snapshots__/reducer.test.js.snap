// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mapEditorReducer returns the initial state 1`] = `
Object {
  "blocksets": Object {
    "primary": Object {
      "blocks": Array [],
      "id": null,
      "palette": Array [],
      "tiles": Array [],
    },
    "secondary": Object {
      "blocks": Array [],
      "id": null,
      "palette": Array [],
      "tiles": Array [],
    },
  },
  "border": Object {
    "block": Array [],
    "collision": Array [],
    "dimensions": Array [
      0,
      0,
    ],
    "height": Array [],
  },
  "canonicalMap": Object {
    "block": Array [],
    "collision": Array [],
    "dimensions": Array [
      0,
      0,
    ],
    "height": Array [],
  },
  "connections": Array [],
  "entities": Array [],
  "linked": false,
  "loading": false,
  "map": Object {
    "block": Array [],
    "collision": Array [],
    "dimensions": Array [
      0,
      0,
    ],
    "height": Array [],
  },
  "scripts": Array [],
}
`;
